src/ui/components/MetadataCard/index.js

```diff
@@ -0,0 +1,34 @@
+/* @flow */
+import React from 'react';
+
+import LoadingText from 'ui/components/LoadingText';
+
+import './styles.scss';
+
+
+type Props = {|
+  metadata: Array<Object>,
+|};
+
+const MetadataCard = (props: Props) => {
+  if (!props.metadata) {
+    throw new Error('The metadata property is required');
+  }
+
+  return (
+    <div className="MetadataCard">
+      {props.metadata.map(({ content, title } = {}) => {
```

Comment:
This is weird no? I think each `metadata` entry should always contain these properties. I don't see where it is missing in the current code.

Reply:
Also I dont find any test cases for that.

Reply:
Agreed, added some tests for that.